adagrams/game.py

```diff
@@ -1,11 +1,142 @@
+import random
+import copy
+
+ # Two Dimensional Array
+
+LETTER_POOL = [
+	['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A'],
+	['B', 'B'], 
+	['C', 'C'], 
+	['D','D','D','D'],
+	['E','E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'], 
+	['F', 'F'], 
+	['G', 'G', 'G'], 
+	['H', 'H'], 
+	['I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I'], 
+	['J'], 
+	['K'], 
+	['L', 'L', 'L', 'L'], 
+	['M', 'M'],
+	['N', 'N', 'N', 'N', 'N', 'N'],
+	['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O'], 
+	['P', 'P'], 
+	['Q'], 
+	['R', 'R', 'R', 'R', 'R', 'R'], 
+	['S', 'S', 'S', 'S'], 
+	['T', 'T', 'T', 'T', 'T', 'T'], 
+	['U', 'U', 'U', 'U'], 
+	['V', 'V'], 
+	['W', 'W'], 
+	['X'], 
+	['Y', 'Y'],
+	['Z']
+]
+
+# helper function to create one big list of strings
+
+def get_one_list_of_letters():
+
+    total_letters_list = []
+    for elem in LETTER_POOL:
+        for char in elem:
+            total_letters_list.append(char)
+    return total_letters_list
+
 def draw_letters():
-    pass
+
+    total_letters_list = get_one_list_of_letters()
+
+    # start with an empty hand and initialize counter at 0
+    hand = []
+    i = 0
+    while i < 10:
+        # randomize letter choice
+        random_letter = total_letters_list[random.randint(0, 97)]
```

Comment:
Small comment, one thing that we want to try to avoid as developers is hard coding values inside for code base. So, instead using `97`, we could maybe do `len(total_letters_list)` that way if anyone wants to change the weights of the letter then they won't have change the `97` as well! Again, just a small change to make your code more scalable.